packages/labs/src/components/timezone-picker/timezone-picker.md

```diff
@@ -0,0 +1,63 @@
+@# TimezonePicker
+
+`TimezonePicker` allows the user to select from a list of timezones.
+
+@reactExample TimezonePickerExample
+
+@## JavaScript API
+
+[Moment Timezone](http://momentjs.com/timezone/) is used internally
+for the list of available timezones and timezone metadata.
```

Comment:
this line belongs at the end of the section, not the beginning: it's an implementation detail, not relevant to usage.